client/src/fluid/FluidTokenizer.res

```diff
@@ -166,6 +167,33 @@ let rec patternToToken = (matchID: id, p: FluidPattern.t, ~idx: int): list<fluid
     Belt.List.concatMany([whole, list{TPatternFloatPoint(matchID, id, idx)}, fraction])
   | PNull(id) => list{TPatternNullToken(matchID, id, idx)}
   | PBlank(id) => list{TPatternBlank(matchID, id, idx)}
+  | PTuple(id, first, second, theRest) =>
+    let subPatterns = list{first, second, ...theRest}
+
+    // CLEANUP: this is likely not the most efficient way to handle this.
```

Comment:
Performance wise, this seems totally fine. As a result I would remove the comment as I think it'll be a red herring. They only performance improvements I would recommend here is to pull out of List.length as it is an invariant.

Reply:
Yeah I agree the perf is fine here - I was more looking for terser code, but again this is fine. :heavy_check_mark: w/r/t List.length